core/tests/protractor_utils/LibraryPage.js

```diff
@@ -77,6 +77,10 @@ var LibraryPage = function() {
     var searchInput = (
       browser.isMobile ? await searchInputs.get(1) :
       await searchInputs.first());
+    await waitFor.visibilityOf(
+      searchInput,
+      'search input taking too long to appear'
+    );
```

Comment:
`action.clear` already waits for the element to be visible, so this is redundant. You can check out its implementation here:

https://github.com/oppia/oppia/blob/96c404b45e61e53d07ca0e4f308b6d330bd49f81/core/tests/protractor_utils/action.js#L30

Reply:
done